File: TypeScript/Codewars/src/resolutions/Basic-Mathematical-Operations.ts
export function basicOp(operation: string, value1: number, value2: number): number {
	const Operations = {
		addition: "+",
		division: "/",
		multiplication: "*",
		subtraction: "-",
	};

	if (operation === Operations.addition) return value1 + value2;
	if (operation === Operations.subtraction) return value1 - value2;
	if (operation === Operations.multiplication) return value1 * value2;
	if (operation === Operations.division) return value1 / value2;

	return 0;
}

File: Csharp/CodeWars/src/Completed/BasicMathematicalOperations.cs
namespace CodeWars.Completed;

public class BasicMathematicalOperations
{
    public static double BasicOp(char operation, double value1, double value2) =>
        operation switch
        {
            '+' => value1 + value2,
            '-' => value1 - value2,
            '*' => value1 * value2,
            '/' => value1 / value2,
            _ => throw new ArgumentException("Operação inválida", nameof(operation))
        };
}
